### PR TITLE
Add new connector types from errata

### PR DESCRIPTION
**Pull request by mdwcrft**
Adding 3 new connector types from the Errata, and any missing descriptions.

Screenshot 2022-04-20 at 21 21 29

### CODE REVIEW

**Comment by mdwcrft** [comment]
See https://github.com/mobilityhouse/ocpp/pull/331#issuecomment-1123906583

File: ocpp/v201/enums.py [hunk 1]
```diff
@@ -371,20 +375,39 @@ class ConnectorType(str, Enum):
...
     s309_3p_16a = "s309-3P-16A"
+    # 32A 3 phase IEC60309 socket
     s309_3p_32a = "s309-3P-32A"
+    # UK domestic socket a.k.a. 13Amp
     s_bs1361 = "sBS1361"
+    # CEE 7/7 16A socket. May represent 7/4 & 7/5 a.k.a Schuko
     s_cee_7_7 = "sCEE-7-7"
+    # IEC62196-2 Type 2 socket a.k.a. Mennekes connector
     s_type2 = "sType2"
+    # IEC62196-2 Type 2 socket a.k.a. Scame
```
Comment (tropxy) [inline ocpp/v201/enums.py:390]: Type 3 and not type 2

**Review by tropxy — CHANGES_REQUESTED** [review]

**Review by tropxy — CHANGES_REQUESTED** [review]

File: ocpp/v201/enums.py (hunk 1 above)
Comment (mdwcrft) [reply ocpp/v201/enums.py]: Typo from spec, fixed here

File: ocpp/v201/enums.py [hunk 2]
```diff
@@ -371,20 +375,39 @@ class ConnectorType(str, Enum):
...
     s309_1p_32a = "s309-1P-32A"
+    # 16A 3 phase IEC60309 socket
     s309_3p_16a = "s309-3P-16A"
+    # 32A 3 phase IEC60309 socket
     s309_3p_32a = "s309-3P-32A"
+    # UK domestic socket a.k.a. 13Amp
     s_bs1361 = "sBS1361"
+    # CEE 7/7 16A socket. May represent 7/4 & 7/5 a.k.a Schuko
     s_cee_7_7 = "sCEE-7-7"
+    # IEC62196-2 Type 2 socket a.k.a. Mennekes connector
```
[the remark this answers is not in the document]
Comment (mdwcrft) [reply ocpp/v201/enums.py]: Typo from spec, fixed here

**Review by tropxy — CHANGES_REQUESTED** [review]
could you also add a test for the ConnectorType enum?

https://github.com/mobilityhouse/ocpp/blob/master/tests/v201/test_v201_enums.py

**Review by tropxy — APPROVED** [review]

**Pull request merged**